src/sgame/sg_admin.cpp

```diff
@@ -2350,6 +2358,92 @@ bool G_admin_setlevel( gentity_t *ent )
 	return true;
 }
 
+bool G_admin_slap( gentity_t *ent )
+{
+	int            pid;
+	double         damage = 0.0f;
+	char           name[ MAX_NAME_LENGTH ], err[ MAX_STRING_CHARS ];
+	gentity_t      *vic;
+	vec3_t         dir;
+
+	if ( level.intermissiontime )
+	{
+		return false;
+	}
+
+	if ( trap_Argc() < 2 )
+	{
+		ADMP( QQ( N_("^3slap:^* usage: slap [^3name|slot#|admin#^7] (^3damage^7)") ) ); 
+		return false; // todo: G_admin_helptext( "slap" ) instead of this copypasta
+	}
+
+	trap_Argv( 1, name, sizeof( name ) );
+
+	if ( ( pid = G_ClientNumberFromString( name, err, sizeof( err ) ) ) == -1 )
```

Comment:
please declare the variable as close as possible from where it's used, it's more readable.
This would be:

```
int pid = G_ClientNumberFromString( name, err, sizeof( err ) ) 
if ( pid == -1 )
```